pyvista/plotting/plotter.py

```diff
@@ -3337,7 +3337,7 @@ def add_mesh(
         # Try to plot something if no preference given
         if scalars is None and color is None and texture is None:
             # Prefer texture first
-            if len(list(mesh.textures.keys())) > 0:
+            if len(list(mesh._textures.keys())) > 0:
```

Comment:
Preexisting logic, but
```suggestion
            if len(mesh._textures) > 0:
```
or even
```suggestion
            if mesh._textures:
```